src/screens/profile-screen/Socials.tsx

```diff
@@ -0,0 +1,42 @@
+import { ProfileUser } from 'audius-client/src/pages/profile-page/store/types'
+import { View } from 'react-native'
+
+import IconInstagram from 'app/assets/images/iconInstagram.svg'
+import IconTwitterBird from 'app/assets/images/iconTwitterBird.svg'
+import { makeStyles } from 'app/styles/makeStyles'
+import { useThemeColors } from 'app/utils/theme'
+
+import { ProfileBadge } from './ProfileBadge'
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  socials: {
+    display: 'flex',
```

Comment:
fyi we do have a `...flexRowCentered()` helper that will do this, might be cool to include that in the things passed to the style creator function

Reply:
yes love this idea, will add them